### DISCUSSION TITLE
Dialog focus and  dom access method

### DISCUSSION THREAD

**Discussion by james-ep01** [clientIO/joint#2840] [Q&A]
### Introduction

I want to focus on the input field in the Dialog and access the field in the Dialog to set the value, but I don't know when it is loaded, so I can't access it.
I can't focus it or access the dom with jquery, etc. What should I do?

Below is an example where I select a white label and modify it in the Dialog.
![image](https://github.com/user-attachments/assets/fbd9caba-a885-4ac2-80ac-d297242c90b7)

Thank you.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
You are responsible for creating the content of the dialog box and opening it.

1. Keep reference to the input when you create the input.
```ts
const inputEl = document.createElement('input');
const dialog = new ui.Dialog({
    content: inputEl
});
dialog.open();
inputEl.value = 5;
```

2. Find the DOM element after you open the dialog.
```ts
dialog.open();
dialog.el.querySelector('input').value = 5;
```

An example of the `ui.Inspector` inside a `ui.Dialog` is showcase here: https://www.jointjs.com/demos/organizational-chart
